modules/inflowwind/src/InflowWind.txt

```diff
@@ -91,11 +91,19 @@ typedef  ^                       ^                 ReKi              HAWC_dy
 typedef  ^                       ^                 LOGICAL           SumPrint          -     -     -     "Write summary info to a file <ROOTNAME>.IfW.Sum"        -
 typedef  ^                       ^                 IntKi             NumOuts           -     -     -     "Number of parameters in the output list (number of outputs requested)" -
-typedef  ^                       ^                 CHARACTER(ChanLen) OutList           :     -     -     "List of user-requested output channels"                 -
-typedef  ^                       ^                 IntKi             SensorType        -  SensorType_None    -     "Sensor type (for lidar/sensor module)"                 -
-typedef  ^                       ^                 IntKi             NumPulseGate      -     -     -     "the number of range gates to return wind speeds at"                 -
+typedef  ^                       ^                 CHARACTER(ChanLen) OutList           :     -     -    "List of user-requested output channels"                 -
+typedef  ^                       ^                 IntKi             SensorType        -     -    -      "Sensor type (for lidar/sensor module)"                 -
+typedef  ^                       ^                 IntKi             NumBeam           -   -    -        "Number of lidar beams"  
+typedef  ^                       ^                 IntKi             NumPulseGate      -     -     -     "The number of range gates to return wind speeds at"                 -
 typedef  ^                       ^                 ReKi              RotorApexOffsetPos 3    -     -     "position of the lidar unit relative to the rotor apex of rotation"	m
 typedef  ^                       ^                 LOGICAL           LidRadialVel      -     -     -     "TRUE => return radial component, FALSE => return 'x' direction estimate"	-
+typedef  ^                       ^                 ReKi              FocalDistanceX    :     -     -     "LIDAR LOS focal distance co-ordinates in the x direction"	m
+typedef  ^                       ^                 ReKi              FocalDistanceY    :     -     -     "LIDAR LOS focal distance co-ordinates in the y direction"	m
+typedef  ^                       ^                 ReKi              FocalDistanceZ    :     -     -     "LIDAR LOS focal distance co-ordinates in the z direction"	m 
+typedef  ^                       ^                 ReKi              PulseSpacing    -       -     -     "Distance between range gates" 	m
+typedef	 ^			                 ^                 ReKi	             MeasurementInterval	- 	-    -	   "Time between each measurement"	s
+typedef  ^                       ^                 ReKi              URefLid        -     -       -      "Reference average wind speed for the lidar"  m/s 
+typedef	^			                   ^                 IntKi	           ConsiderHubMotion	-	   -  	-	     "Flag whether the hub motion's impact on the Lidar measurement will be considered"	-
```

Comment:
More column alignment

Reply:
I've fixed the column alignment throughout the .txt files